src/main/java/lotto/dto/FinalResultResponse.java

```diff
@@ -0,0 +1,19 @@
+package lotto.dto;
+
+import lotto.domain.prize.FinalResults;
+
+import java.math.BigDecimal;
+import java.text.DecimalFormat;
+
+import static lotto.view.constants.PrintFormat.SEPARATOR_WITH_POINT_FORMAT;
+
+public record FinalResultResponse(
+        BigDecimal yield,
+        FinalResults finalResults
+) {
+    private static final DecimalFormat YIELD_FORMAT = SEPARATOR_WITH_POINT_FORMAT.getFormat();
+
+    public String getFormattedYield() {
+        return YIELD_FORMAT.format(yield);
+    }
```

Comment:
[이렇게 바꿔보면 좋을 것 같아요!]

```suggestion
    public String getFormattedYield() {
        return SEPARATOR_WITH_POINT_FORMAT.format(yield);
    }
```
PrintFormat enum을 활용해서 문자열로 만들어준다면 해당 format 메서드가 PrintFormat에 위치하면 좋겠네요.

enum을 상수 집합으로 활용하는 방법 + 공통적인 기능을 모아두면 응집도가 올라갈 것 같아요 :) (개인적인 생각입니다)

Reply:
말씀주신 방법대로 활용하면, 응집도를 훨씬 높일 수 있겠어요!
Enum을 바탕으로 바로 포맷팅 하는 메소드를 호출하는 것!
좋은 아이디어 얻고 갑니다 :)